cmd/bucket-notification-handlers.go

```diff
@@ -83,6 +83,7 @@ func (api objectAPIHandlers) GetBucketNotificationHandler(w http.ResponseWriter,
 		nConfig = &event.Config{}
```

Comment:
Can `nConfig = &event.Config{}` be changed to `nConfig = &event.Config{XMLNS : "http://s3.amazonaws.com/doc/2006-03-01/"}` ?

Reply:
Move this added code to `ParseConfig` function in `config.go` https://github.com/minio/minio/blob/master/pkg/event/config.go#L279

Reply:
`SetBucketNofication.java` is failing. It looks like something that needs to be fixed on the `minio-java` side though